### ISSUE TITLE
Change Incident Status on Reply works on Closed Work Items

### ISSUE THREAD

**Issue by AdhocAdam** [AdhocAdam/smletsexchangeconnector#115]
**Describe the bug**
If enabled, the feature to change Incident Status on Reply works against Closed Work Items.

**Help us reproduce the bug**
Version of the connector you are using: 1.4.6
Features/variables you have enabled (i.e. $enableAzureCognitiveServicesForNewWI): $changeIncidentStatusOnReply = $true

**Expected behavior**
Replies shouldn't impact the status of a Closed Work Items as this can skew metrics.

**Location and Environment**
Where are you running this from? Task Scheduler? SMA? Workflow server?: Task Scheduler
What SCSM version are you running? 2016

**Additional context**

**Comment by AdhocAdam** [comment]
**workaround**: The condition that checks for this can be modified from...

```powershell
if ($changeIncidentStatusOnReply)
```

to 

```powershell
if (($changeIncidentStatusOnReply) -and ($workitem.Status.DisplayName -ne "Closed"))
```

**Comment by AdhocAdam** [comment]
Included in #113
